decidim-core/spec/presenters/decidim/push_notification_presenter_spec.rb

```diff
@@ -26,6 +26,13 @@ module Decidim
           # rubocop:enable RSpec/AnyInstance
         end
       end
+
+      describe "#url" do
+        it "returns the url to the comment" do
+          # comment urls are anchors from the resource page to the comment
+          expect(subject.url).to include("#comment_#{extra[:comment_id]}")
+        end
+      end
     end
```

Comment:
I prefer to have a regression spec that give me the exact same exception from the original bug report, just to be 100% sure that we're solving this same problem with at least one of the models from the report. This is what I had in mind, feel free to improve on it: 

```suggestion
    end

    context "when it's a blog post" do
      let(:event_class) { Decidim::Blogs::CreatePostEvent }
      let(:event_name) { "decidim.events.blogs.post_created" }
      let(:resource) { create(:post) }
      let(:notification) { create(:notification, event_class:, event_name:, resource:) }

      describe "#body" do
        it "returns text without links and with HTML entities unescaped" do
          expect(subject.body).to eq("The post #{translated(resource.title)} has been published in #{translated(resource.component.participatory_space.title)}")
        end
      end

      describe "#url" do
        it "returns the url to the post" do
          expect(subject.url).to eq(resource_locator(resource).url)
        end
      end
    end
```

You should change the title of the first case ("with a valid notification" should be "when it's a comment" so it's consistent).